src/ui/components/ExplorerHome/ExplorerTimeline.js

```diff
@@ -29,7 +29,6 @@ const ExplorerTimeline = (props: ExplorerTimelineProps): ReactNode => {
   const width = props.width || 300;
   const height = props.height || 25;
   const viewBox = `0 0 ${width} ${height}`;
-
   let grainValues;
   if (grainExists) {
     const grain = props.timelines.grain || [];
```

Comment:
this too?

Reply:
`grainValues` is actually what potentially would need this re-instantiation, but it doesn't since it is derived from a map operation on the `grain` prop.